src/nextjs/server/index.tsx

```diff
@@ -302,6 +303,21 @@ async function isAuthenticated(token: string | null): Promise<boolean> {
   if (!token) {
     return false;
   }
+
+  // First, try using the JWKS from the environment variable to do token
+  // verification locally (networkless mode).
+  try {
+    const envJwks = process.env.CONVEX_AUTH_JWKS
+    if (envJwks) {
+      const jwkSet = createLocalJWKSet(JSON.parse(envJwks))
+      const verifiedToken = await jwtVerify(token, jwkSet);
+      return !!verifiedToken.payload.sub;
+    }
+  } catch (error: any) {
+    console.error("Error verifying token", error);
```

Comment:
```suggestion
    console.error("Error verifying token using process.env.CONVEX_AUTH_JWKS", error);
```
(we could also say something like "make sure this matches the value set in your Convex `JWKS` environment variable")